src/main/groovy/com/deploygate/gradle/plugins/DeployGate.groovy

```diff
@@ -98,6 +105,37 @@ class DeployGate implements Plugin<Project> {
         }
     }
 
+    private void createAABAwareUploadTasks(Project project, ApkInfo apkInfo) {
+        if (!AndroidPlatformUtils.isAppBundleSupported()) {
+            return
+        }
+
+        def deployTarget = project.deploygate.apks.findByName(apkInfo.variantName) as DeployTarget
+        def tasksDependsOn = project.getTasksByName("loginDeployGate", false).toList()
+        def bundleTask = project.getTasksByName("bundle${apkInfo.variantName.capitalize()}", false)
+
+        if (!bundleTask.empty && !deployTarget?.bundle?.skipBundle) {
```

Comment:
In the future, it will be required to avoid confusion if we will support aab direct upload. That's why I've added this.

But as you said, it's enough for now 🤔and my thought was changed!
I'll rename the task name to `uploadDeployGateFromAab` and then `skipBundle` is not needed and understandable! Thanks!

Reply:
... I believe nobody uses product flavor `from` and build type `aab`... 🙏